README.md

```diff
@@ -159,6 +159,207 @@ python ./scripts/train_hf_ner.py ./data/annotations/train.spacy ./data/annotatio
+prominently in the UI. We can press the <kbd>ACCEPT</kbd> (check mark) button to
+include the text as a positive example or press the <kbd>REJECT</kbd> (cross
+mark) button if it is a negative example.
+
+
+```sh
+python -m prodigy textcat.openai.correct my_textcat_data data/reddit_r_cooking_sample.jsonl \
+    --labels recipe \
+    --verbose \
+    -F recipes/openai_textcat.py
+```
+
+<img src="https://user-images.githubusercontent.com/12949683/214230166-ee492fe5-04da-4b93-9590-b5ef23ce488d.png" width="600"/>
+
+
+#### Multilabel and multiclass text categorization
+
+Now, suppose we want to classify Reddit comments as a recipe, a feedback, or a
+question. We can write the following prompt: 
+
+```
+Classify the text below to any of the following labels: recipe, feedback, question.
+The task is exclusive, so only choose one label from what I provided.
+
+Your answer should only be in the following format:
+answer: <string>
+reason: <string>
+
+Text:
+```
+
+Then, we can use this recipe to handle multilabel and multiclass cases by
+passing the three labels to the `--labels` parameter. We should also set the
+`--exclusive-classes` flag to render a single-choice UI:
+
+```sh
+python -m prodigy textcat.openai.correct my_textcat_data data/reddit_r_cooking_sample.jsonl \
+    --labels recipe,feedback,question \
+    --verbose \
+    --exclusive-classes \
+    -F recipes/openai_textcat.py
+```
+
+<img src="https://user-images.githubusercontent.com/12949683/214230269-bdacbbc8-8edc-4be5-8334-0b7eaf4712e0.png" width="600" />
+
+#### Writing templates
+
+We write these prompts as a .jinja2 template that can also take in examples for
+few-shot learning. You can create your own [template](templates) and provide it
+to the recipe with the `--prompt-path` or `-p` option.  Additionally, with
+`--examples-path` or `-e` you can set the file path of a .y(a)ml or .json file
+that contains additional examples:
+
+```bash
+python -m prodigy textcat.openai.correct my_textcat_data \
+    ./data/reddit_r_cooking_sample.jsonl \
+    --labels recipe \
+    --prompt-path ./templates/textcat_prompt.jinja2 \
+    --examples-path ./examples/textcat_binary.yaml -n 2 \
+    -F ./recipes/openai_textcat.py
+```
+
+Similar to the NER recipe, this recipe also converts the predictions into an 
+annotation task that can be rendered with Prodigy. For binary classification, we
+use the [`classification`](https://prodi.gy/docs/api-interfaces#classification)
+interface with custom HTML elements, while for multilabel or multiclass text
+categorization, we use the
+[`choice`](https://prodi.gy/docs/api-interfaces#choice) annotation interface.
+Notice that we include the original prompt and ChatGPT's response in the UI. 
+
+Lastly, you can use the `--verbose` or `-v` flag to show the exact prompt and
+response on the terminal. Note that because the requests to the API are batched,
+you might have to scroll back a bit to find the current prompt.
+
```

Comment:
Nitpick, but maybe after labelling some examples we can mention that you can train a model? 

```bash
# Depending on the setting, either one of these?

# If multi-class
python -m prodigy train --textcat my_textcat_data

# If multi-label
python -m prodigy train --textcat-multilabel my_textcat_data
```